src/django/api/models.py

```diff
@@ -139,13 +139,13 @@ class User(AbstractBaseUser, PermissionsMixin):
     updated_at = models.DateTimeField(auto_now=True)
 
     def __str__(self):
-        return self.name
+        return self.username or ''
```

Comment:
Since we're using email addresses for usernames -- and `username` will theoretically always be unset -- would it make sense for these to be `self.email`?

Reply:
I was concerned about spreading the email address around but after thinking about it I don't think that concern is warranted. We should only be implicitly converting `User` to string in shell or admin context, so email will be more helpful.

Reply:
Rebased with this change.